src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
I think the idea/goal here is that the "System.Memory" package can be used from any runtime that supports .NET Standard 1.0, which is pretty much every platform anyone cares about. This lets you write a portable library using `Span`, etc. As an optimization, there is a special implementation for `netcoreapp2.0` that happens to be much faster.